pkgs/by-name/mu/multiqc/package.nix

```diff
@@ -0,0 +1,116 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  procps,
+  versionCheckHook,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "multiqc";
+  version = "1.26";
+
+  disabled = python3Packages.pythonOlder "3.8";
+
+  # Two data sources. One for the code, another for the test data
+  srcs = [
+    (fetchFromGitHub {
+      name = "multiqc";
+      owner = "MultiQC";
+      repo = "MultiQC";
+      rev = "v${version}";
+      hash = "sha256-MPAw6gG/3LzdskkDXOTDEM1NpG0sH9GvklYFQ1ZXWIs=";
+    })
+    (fetchFromGitHub {
+      owner = "MultiQC";
+      repo = "test-data";
+      rev = "67435083a8bfa228dca3dda7d835facef15fc2c7";
+      hash = "sha256-oYmPIJSy6dOKPcMr3B4foGoWcerA29x0XeGoU4dSYsA=";
+      name = "test-data";
+    })
+  ];
+
+  sourceRoot = "multiqc";
+
+  # Some tests run subprocess.run() with "ps" and multiqc""
+  postPatch = ''
+    substituteInPlace multiqc/plots/plotly/plot.py \
```

Comment:
I think this can be removed by adding ps to the nativeBuildInputs (to be tested)

Reply:
Great suggestion, it works !